benchmark/utils/DriverUtils.swift

```diff
@@ -413,9 +399,7 @@ func runBench(_ test: Test, _ c: TestConfig) -> BenchResults? {
       if c.verbose {
         print("    Measuring with scale \(scale).")
       }
-      test.setUpFunction?()
       elapsed_time = sampler.run(test.name, fn: testFn, num_iters: scale)
```

Comment:
Same here. Why are you changing this in this PR. It should be a separate PR with reasoning.